#goal definition
string objId
bool planOnly
---
#result definition
moveit_msgs/MoveItErrorCodes error_code
int32 cost
---
#feedback
int32 status
